Moq.Dapper.Test/DapperExecuteAsyncTest.cs

```diff
@@ -12,15 +14,70 @@ public void ExecuteAsync()
         {
             var connection = new Mock<DbConnection>();
 
-            connection.SetupDapperAsync(c => c.ExecuteAsync("", null, null, null, null))
+            connection.SetupDapperAsync(c => c.ExecuteAsync(It.IsAny<string>(), null, null, null, null))
```

Comment:
This is better, but I'll revert it for not just to reduce the surface of changes together with result -> actual.